src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke.command;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import static duke.command.Command.*;
+
+import java.util.ArrayList;
+
+public class Duke {
+    public static void main(String[] args) {
+
+        printWelcomeMessage();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        try {
```

Comment:
Consider refactoring down read and write inside the main into their own methods to reduce long methods greater than 30 lines of code.

Code Quality Guideline: Avoid Long Methods